tests/test_csc.py

```diff
@@ -37,11 +37,17 @@
 
 
 class Harness:
-    def __init__(self, initial_state, config_dir=None):
+    """Make a Watcher CSC and a remote for it.
+
+    Parameters
+    ----------
+    config_dir : `str` (optional)
```

Comment:
There's also an `initial_state` parameter. I think this is all stuff we should lift into that "how to write CSC tests" doc.

Reply:
I actually ditched that parameter as it is not practical to start the Watcher in anything other than STANDBY (because it needs a config file).